webapp/app/[locale]/tunnel/transaction-history/_components/transactionHistory.tsx

```diff
@@ -26,9 +30,14 @@ const useTransactionsHistory = function (filter: FilterOptions) {
         .concat(withdrawals)
         .filter(
           operation =>
-            filter === 'all' ||
-            (filter === 'bitcoin' && isBtcOperation(operation)) ||
-            (filter === 'ethereum' && isEvmOperation(operation)),
+            filter.operation === 'all' ||
+            (filter.operation === 'bitcoin' && isBtcOperation(operation)) ||
+            (filter.operation === 'ethereum' && isEvmOperation(operation)),
+        )
+        .filter(
+          operation =>
+            filter.action === 'all' ||
+            (filter.action === 'pending' && isPendingOperation(operation)),
```

Comment:
it's missing filtering by `filter.type` (either `deposits` or `withdrawals`)

Reply:
fixed__